### DISCUSSION TITLE
[CV-Site] - Athanasopoulou Konstantina

### DISCUSSION THREAD

**Discussion by kwnnath** [upatras-hci/iv#36] [Show and tell]
### CV-site
<p align="justify">
Το συγκεκριμένο παραδοτέο στόχευε στη δημιουργία του βιογραφικού μας με χρήση του συστήματος Jekyll. Αρχικά, πρόσθεσα ένα νέο αποθετήριο με όνομα "jekyll-cv". Εκεί δημιούργησα τον φάκελο _data στον οποίο πρόσθεσα τα αρχεία yaml με τα προσωπικά μου στοιχεία. 
</p>

- **[ο σύνδεσμος του αποθετηρίου](https://github.com/kwnnath/jekyll-cv)** 
- **[ο σύνδεσμος του site](https://kwnnath.github.io/jekyll-cv/)**

**Comment by e-liouta** [comment]
Πολύ ωραίο!

**Comment by kwnnath** [comment]
Ευχαριστώ πολύ!
